src/components/common/form/InputRounded.js

```diff
@@ -15,8 +15,8 @@ import ErrorText from './ErrorText'
  */
 const InputRounded = ({ styles, containerStyle, theme, icon, iconSize, iconColor, error, onChange, ...inputProps }) => {
   const handleChange = useCallback(
-    event => {
-      onChange(event.target.value)
```

Comment:
@serdiukov-o-nordwhale @MathiasMoralesDixtra @mauriguz dont we need native/web separation here?

Reply:
This works for both, I tested on the web